ytfzf

```diff
@@ -150,6 +150,10 @@ trending_tab="${trending_tab-}"
 sort_name="${sort_name-}"
 #to detach the video player from the terminal
 detach_player=${detach_player-0}
+
+# in loop mode the player always needs to detach
+[ "$enable_loop" -eq 1 ] && detach_player=1
```

Comment:
add the `detach_player=1` part in the `parse_opt` function where `-l` is as well as here.